lib/colocated-broccoli-plugin.js

```diff
@@ -150,7 +150,10 @@ module.exports = class ColocatedTemplateProcessor extends Plugin {
           }
         );
 
-        if (hasTemplate && !jsContents.includes('export default')) {
+        if (hasTemplate && jsContents.includes('export { default }')) {
+          jsContents = jsContents.replace('export { default }', "import Component");
```

Comment:
my only reluctance here is if someone potentially already defined `Component` for some reason.

It may be safer to rename `Component` (since it's just a default import), to `__Component` or something?

Reply:
Good point, will change :+1: Not too sure if that is actually possible, because not changing any of the JS contents didn't work in practice for me. So like:
```js
import { hbs } from 'ember-cli-htmlbars;
const __COLOCATED_TEMPLATE__ = ...;
export { default } from '...';
```
did not work.

Reply:
I have changed it :+1: